src/mgmt/tests/mocks.c

```diff
@@ -263,6 +263,57 @@ ds_mgmt_free_pool_list(struct mgmt_list_pools_one **poolsp, uint64_t len)
 	}
 }
 
+/*
+ * Mock ds_mgmt_pool_list_cont
+ */
+int				 ds_mgmt_pool_list_cont_return;
+struct daos_pool_cont_info	*ds_mgmt_pool_list_cont_out;
+uint64_t			 ds_mgmt_pool_list_cont_nc_out;
+
+int ds_mgmt_pool_list_cont(uuid_t uuid,
+			   struct daos_pool_cont_info **containers,
+			   uint64_t *ncontainers)
+{
+	size_t i;
+
+	if (containers != NULL && ncontainers != NULL &&
+	    ds_mgmt_pool_list_cont_out != NULL) {
+		*ncontainers = ds_mgmt_pool_list_cont_nc_out;
+		D_ALLOC_ARRAY(*containers, *ncontainers);
+		for (i = 0; i < *ncontainers; i++) {
+			uuid_copy((*containers)[i].pci_uuid,
+				  ds_mgmt_pool_list_cont_out[i].pci_uuid);
+		}
```

Comment:
Should be okay to just bulk memcpy the whole array, right? Since it is the same struct and the UUIDs are just byte arrays, not dynamically allocated.

Reply:
I missed this comment when updating the patch. If you want I can update the patch to do a single mempy.

Reply:
Had to merge master and make other changes, so replacing the loop with memcpy